project-service/src/models/index.js

```diff
@@ -3,7 +3,7 @@ import path from 'path';
 import Sequelize from 'sequelize';
 import configs from '../config.js';
 
-let env = process.env.NODE_ENV || 'development';
+let env = process.env.NODE_ENV || 'test';
```

Comment:
So what would the environment be if i do docker-compose? `production`? How do I run on `development` database?